src/layout/page-layout.scss

```diff
@@ -0,0 +1,384 @@
+// stylelint-disable max-nesting-depth
+// stylelint-disable selector-max-specificity
+// stylelint-disable no-duplicate-selectors
+
+$Layout-divider-color: var(--color-border-default) !default;
+$Layout-responsive-variant-max-breakpoint: 'md' !default;
+
+:root {
+  --Layout-pane-width: #{map-get($sidebar-width, 'sm')};
```

Comment:
I _love_ this use of custom properties as a config!

Reply:
:octocat: 💌